synapse/handlers/identity.py

```diff
@@ -595,10 +607,10 @@ async def _lookup_3pid_v1(self, id_server, medium, address):
         """Looks up a 3pid in the passed identity server using v1 lookup.
 
         Args:
-            id_server (str): The server name (including port, if required)
+            id_server: The server name (including port, if required)
                 of the identity server to use.
-            medium (str): The type of the third party identifier (e.g. "email").
-            address (str): The third party identifier (e.g. "foo@example.com").
+            medium: The type of the third party identifier (e.g. "email").
+            address: The third party identifier (e.g. "foo@example.com").
```

Comment:
ok, I'm going to stop saying "ditto" now. What am I missing here?

Reply:
Hmm...I'm not sure why I did that, sorry to waste time! I'll take another look.